pkg/azuredisk/azuredisk_v2.go

```diff
@@ -89,6 +93,25 @@ const (
 	azDriverNodeHealthy      azDriverNodeStatus = "Driver node healthy."
 )
 
+// LatencyAdapter implements LatencyMetric.
+type LatencyAdapter struct {
+	metric *metrics.HistogramVec
+}
+
+// Observe increments the request latency metric for the given verb/URL.
+func (l *LatencyAdapter) Observe(_ context.Context, verb string, u url.URL, latency time.Duration) {
+	l.metric.WithLabelValues(verb, u.String()).Observe(latency.Seconds())
```

Comment:
We should check if `latency < longThrottleLatency`, where `longThrottleLatency = 50 * time.Millisecond`, and only observe the rate limiter latency that throttled.

Reply:
I think the check would be sign should be reversed here. We want to observe the requests that are above 50ms, so when `latency > longThrottleLatency`.

Reply:
btw, I just checked the code, and all the other calls get throttled too, it is just under 50ms.